HTCondor COBalD/TARDIS docker image (#185)
MatterMiners/tardis: docs/source/index.rst

        This path must contain a valid `cobald.yml` file. The persistent database will then be stored on the local machine.

Running your instance using HTCondor as overlay batch system in Docker
======================================================================

.. content-tabs:: left-col

    For your convenience and for systems without pre-compiled HTCondor support a ready to use docker container is
    provided via Dockerhub.

    You can configure ``COBalD``/``TARDIS`` as described in this documentation and bind mount the directory containing
    the configuration into the containers `/srv` directory.

    .. note::

        This path must contain at least a valid `cobald.yml` file.

.. content-tabs:: right-col

    .. code-block::

        docker run -v $PWD/configuration:/srv matterminers/cobald-tardis-htcondor:latest

HTCondor Token Support
----------------------

.. content-tabs:: left-col

    Starting with the production release series 9.0, HTCondor introduces a new security configuration, which is no
    longer host-based. The security configuration is now user-based and requires authentication to access the HTCondor

>     pool. This is also true for read-only operations like calling `condor_status`. Therefore, this docker images

Reviewer: ```suggestion
    pool. This is also true for read-only operations such as `condor_status`. Therefore, this docker image
```
